boards/thingy91/include/periph_conf.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2023 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_thingy91
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the Thingy:91
+ *
+ * @author      Thanh-Viet Nguyen <topologicrose@protonmail.com>
+ *
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+#include "cfg_clock_32_1.h"
+#include "cfg_rtt_default.h"
+#include "cfg_timer_default.h"
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    UART configuration
+ * @{
+ */
+static const uart_conf_t uart_config[] = {
+    {
+        .dev        = NRF_UARTE0_S,
+        .rx_pin     = GPIO_PIN(0, 2),
+        .tx_pin     = GPIO_PIN(0, 3),
+#ifdef MODULE_PERIPH_UART_HW_FC
+        .rts_pin    = GPIO_UNDEF,
+        .cts_pin    = GPIO_UNDEF,
+#endif
+        .irqn       = UARTE0_SPIM0_SPIS0_TWIM0_TWIS0_IRQn,
+    },
+};
+
+#define UART_0_ISR          (isr_uarte0_spim0_spis0_twim0_twis0) /**< UART0_IRQ */
+
+#define UART_NUMOF          ARRAY_SIZE(uart_config) /**< UART confgiguration NUMOF */
+/**
```

Comment:
I see you have several I2C based sensors but no I2C configuration yet

```suggestion
/**
 * @name    I2C configuration
 * @{
 */
static const i2c_conf_t i2c_config[] = {
    {
        .dev = NRF_TWIM1,
        .scl = GPIO_PIN(0, ???),
        .sda = GPIO_PIN(0, ???),
        .speed = I2C_SPEED_NORMAL
    }
};
#define I2C_NUMOF           ARRAY_SIZE(i2c_config)
/** @} */

/**
```